packages/http-client-csharp/eng/pipeline/publish.yml

```diff
@@ -8,6 +8,12 @@ trigger:
 
 pr: none
 
+parameters:
```

Comment:
We can remove this parameter since it will only work if we are publishing to public npm and we don't want to make that configurable.